appeals/web/src/server/lib/nunjucks-template-builders/gds-components-properties-jsdoc.js

```diff
@@ -0,0 +1,717 @@
+// Multipurpose Definitions (used in multiple components)
+/**
+ * @typedef TextProperty
+ * Use for Text
+ * @type {object}
+ * @prop {string} text Text to use
```

Comment:
These should ideally use the full `property` syntax for consistency, as mentioned elsewhere